Python/compile.c

```diff
@@ -150,6 +150,15 @@ location_is_after(location loc1, location loc2) {
              (loc1.col_offset > loc2.end_col_offset));
 }
 
+static inline bool
+same_location(location a, location b)
+{
+    return a.lineno == b.lineno && 
```

Comment:
According to git diff you have a trailing space here. Surprised CI didn't catch it?

Reply:
Wow, nice catch! I'll fix it.